doc/user_guide.md

```diff
@@ -0,0 +1,1518 @@
+2. The second one contains ~120,000 public ZTF alerts from July 6, 2020. The auxiliary information contains a ~100-day 
+history of detections (limited to reduce the test database size), cross-matches with external catalogs, and a few 
+additional quantities.
+
+We recommend to begin exploring filtering on the first database and then move onto the second one.
+
+In this tutorial, we will show how to use a tool called [MongoDB Compass](https://www.mongodb.com/try/download/compass) 
+(the full version is now free) to construct and debug aggregation pipelines (aka alert filters) using the two cloud
+databases that can be then plugged into `Fritz`.
+
+Filters will be managed on a dedicated page on `Fritz`. A detailed description of the interface and its capabilities
+(including, for example, filter versioning and diff'ing) will be covered elsewhere -- please stay tuned.
+
+##### MongoDB Compass
+
+Download and install MongoDB Compass for your system from [here](https://www.mongodb.com/try/download/compass).
+
+The connection string to access the first sample public alert database:
+```
+mongodb://ztf:FritzZwicky@fritz-test-shard-00-00-uas9d.gcp.mongodb.net:27017,fritz-test-shard-00-01-uas9d.gcp.mongodb.net:27017,fritz-test-shard-00-02-uas9d.gcp.mongodb.net:27017/test?authSource=admin&replicaSet=fritz-test-shard-0&readPreference=primary&appname=MongoDB%20Compass&ssl=true
+```
+
+The connection string to access the second sample public alert database:
+```
+mongodb+srv://ztf:FritzZwicky@fritz-public-20200706.uas9d.gcp.mongodb.net/kowalski
+```
+
+Upon connection:
+- Select the `kowalski` database
+- Select the `ZTF_alerts` collection
+- Go to the Aggregations tab
+- Click the down arrow located to the left of the "Collation" button and select "New Pipeline From Text"
+- Copy paste the contents of the provided JSON files with the sample filters (see below)
+
+Note that Compass shows the output of all stages interactively and also displays/explains any errors in the code, 
+which is extremely helpful for debugging and experimentation.
+
+#### "Upstream" aggregation pipeline stages
+
+[OPTIONAL READ]
+
+The upstream "massaging" mentioned above is performed by `Fritz` for each alert and includes:
+
+- Selecting the newly ingested alert from the `ZTF_alerts` collection by its `candid`
+- Removing the image cutouts to reduce traffic
+- Joining the alert by its `objectId` with the corresponding entry in the `ZTF_alerts_aux` collection, which contains 
+the cross-matches, ML scores, computed quantities, and archival photometry / detection history
+
+The upstream stages also take care of the ACLs.
+
+`Fritz` uses the following four stages: 
+
+1. The first [`$match`](https://docs.mongodb.com/manual/reference/operator/aggregation/match/) 
+stage selects the alert by its candid and ensures the ACLs are respected. 
+
+For example, for a program that has access to the partnership data:
+
+```json
```

Comment:
This kind of step-by-step breakdown of the upstream stages is really nice. I think something similar is needed for a quickstart section, but something that illustrates an alert filter rather than a photometry extractor. Could be something like, filter for programID 1 and 2 data that looks for things that are redder than g-r > 1.